app/pages/shows/_slug/index.vue

```diff
@@ -132,6 +132,7 @@ export default {
         return showslist
           .filter(show => show.play_date < this.getToday)
           .filter(show => show.archived === true)
+          .sort((a, b) => new Date(b.number) - new Date(a.number))
           .sort((a, b) => new Date(b.play_date) - new Date(a.play_date))
```

Comment:
@baxgas if I understand correctly, this will first sort on the item ID and then on the play date? So for those items with the same date, the order will not change (after the first sort), but if the dates are different, the second sort may reshuffle the list, right?

Reply:
@gammaw id cannot be changed, the number can, that was the reason of my choice, so if you eg. want to change the order after upload, you can by editing the number. the id is not flexible (what if you want to put a new item in the middle of the list)
the play_date should overwrite this, so the different date episodes are sorted after in the correct order

Reply:
Ok, thanks. I see. Number or ID, the principle is the same. Only it should be the opposite order, it seems :)

Reply:
> Only it should be the opposite order, it seems :)

Sorry, no. The higher number is in front, that's how we do it. All good.